crates/bevy_internal/src/lib.rs

```diff
@@ -44,6 +44,7 @@ pub use bevy_log as log;
 pub use bevy_math as math;
 #[cfg(feature = "bevy_pbr")]
 pub use bevy_pbr as pbr;
+pub use bevy_picking as picking;
```

Comment:
At a glance, I think this will resolve the CI issue.
```suggestion
#[cfg(feature = "bevy_picking")]
pub use bevy_picking as picking;
```